### DISCUSSION TITLE
Noob question.The command was not found or executable: lake

### DISCUSSION THREAD

**Discussion by 1854039** [lean-dojo/ReProver#74] [Q&A]
I train the retrieval model according to readme step, and I save the deepspeed checkpoint ,
Then I provide the checkpoint as retrieval  when training the generate model. But I get this:
<img width="928" alt="6710d949c240871bf3b5eeaae5f66ff" src="https://github.com/user-attachments/assets/cc0922e2-43d4-4a71-b752-fb4c093a39e2" />
It would be so nice if you could share some kindly idea on why I meet this. Maybe I have to download elan?

**Comment by 1854039** [comment]
the error starts here:
<img width="954" alt="d8f7b47c2731c2cee8a6af06803809d" src="https://github.com/user-attachments/assets/8d57688a-da97-474b-8ba6-0763adabf8db" />
when searching for proof .
 but I can't see the relationship between these 2 parts of error info.

**Comment by yangky11** [comment]
Did you install elan following the README?

https://github.com/lean-dojo/ReProver?tab=readme-ov-file#requirements

> 5. Satisfy the requirements of [LeanDojo](https://github.com/lean-dojo/LeanDojo#requirements).

**Reply by 1854039** [reply]
Yes that's the reason. Thank you so much!
